docker-compose.yml

```diff
@@ -101,59 +63,34 @@ x-checkpointopts: &checkpointopts
   CMAKE_BUILD_TYPE: ${BUILD_TYPE:-release}
   MAGISTRATE_SERIALIZATION_ERROR_CHECKING_ENABLED: ${MAGISTRATE_SERIALIZATION_ERROR_CHECKING:-0}
   CODE_COVERAGE: ${CODE_COVERAGE:-0}
-  https_proxy: ${PROXY-}
-  http_proxy: ${PROXY-}
+  https_proxy: ${PROXY:-}
+  http_proxy: ${PROXY:-}
   CODECOV_TOKEN: ${CODECOV_TOKEN:-}
 
 services:
   ##############################################################################
-  # C++ builds of Checkpoint on linux platform from container baseline
-  # Ubuntu gcc-8 debug build:
-  #   docker-compose run -e CMAKE_BUILD_TYPE=debug ubuntu-cpp
-  ubuntu-cpp:
-    image: ${REPO}:wf-${ARCH}-ubuntu-${UBUNTU}-${COMPILER}-cpp
-    ulimits: &ulimits
-      core: ${ULIMIT_CORE}
-    environment:
-      <<: [*ccache, *checkpointopts]
-    volumes: &ubuntu-volumes
-      - .:/checkpoint:delegated
-      - ${CACHE}${ARCH}-ubuntu-${UBUNTU}-${COMPILER}-cache:/build:delegated
-    command: &vt-cpp-command >
-      /bin/bash -c "
-        /checkpoint/ci/build_cpp.sh /checkpoint /build &&
-        /checkpoint/ci/test_cpp.sh  /checkpoint /build"
-
-  ubuntu-cpp-vt:
-    image: ${REPO}:wf-${ARCH}-ubuntu-${UBUNTU}-${COMPILER}-cpp
-    ulimits: *ulimits
-    environment:
-      <<: [*ccache, *checkpointopts]
-    volumes: *ubuntu-volumes
-    command: &build-against-vt-cpp-command >
-      /bin/bash -c "
-        /checkpoint/ci/build_against_vt.sh /vt /build /checkpoint &&
-        /vt/ci/test_cpp.sh /vt /build"
-
-  ##############################################################################
-  # Interactive C++ setup of Checkpoint on ubuntu platform from container
+  # Interactive C++ setup of magistrate on ubuntu platform from container
   # baseline.
+  # Specify the CACHE variable so Docker Compose knows where to store the build volume.
   #
   # After running:
   #   docker-compose run ubuntu-cpp-interactive
   #
   # You will get a command line where you can run the build command:
-  #   $ /checkpoint/ci/build_cpp.sh /checkpoint /build
-  #   $ /checkpoint/ci/test_cpp.sh  /checkpoint /build
+  #   $ /magistrate/ci/build_cpp.sh /magistrate /build
+  #   $ /magistrate/ci/test_cpp.sh  /magistrate /build
   ubuntu-cpp-interactive:
     image: ${REPO}:wf-${ARCH}-ubuntu-${UBUNTU}-${COMPILER}-cpp
-    ulimits: *ulimits
+    ulimits: &ulimits
+      core: ${ULIMIT_CORE}
     environment:
-      <<: [*ccache, *checkpointopts]
-    volumes: *ubuntu-volumes
+      <<: [*ccache, *magistrateopts]
+    volumes: &ubuntu-volumes
+      - .:/magistrate:delegated
+      - ${CACHE}${ARCH}-ubuntu-${UBUNTU}-${COMPILER}-cache:/build:delegated
 
   ##############################################################################
-  # Build documentation for Checkpoint in the container on ubuntu platform from
+  # Build documentation for magistrate in the container on ubuntu platform from
   # container baseline.
   ubuntu-docs:
```

Comment:
We want to migrate docs build to bake too, now might be the good time to do it.

Note that the documentation is currently failing on `develop` with:
```
 + git config --global user.email jliffla@sandia.gov
+ git config --global user.name 'Jonathan Lifflander'
+ git add checkpoint_docs
+ git commit --allow-empty -m 'Update magistrate_docs (auto-build)'
[master 9f3c37c] Update magistrate_docs (auto-build)
 1 file changed, 2 insertions(+), 2 deletions(-)
 rewrite checkpoint_docs/html/searchdata-v2.js (97%)
+ git push origin master
fatal: could not read Password for 'https://***@github.com': No such device or address
```